CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 Nothing so far
 
+## 7.1.0 - 2022-12-07
```

Comment:
We moeten nog wel even bedenken of het handig is om dit in versie 7 te mergen, er zijn bijv. in deze bundle nog wel wat bugfixes geweest t.a.v. Symfony 5. Als we nieuwe bugfixes nodig hebben word je nu geforceerd om ook doctrine/dbal te upgraden in je project...

Reply:
Het lijkt me niet handig om dit in een minor versie te doen nee. Is er geen dbal v 2.x versie die forward compatible is?

Reply:
Zowel dbal 2 als 3 toestaan lijkt goed te werken op de plekken waar ik denk dat de code wordt afgespeeld die ik heb aangepast. Ook heb ik hier en daar nog wat `execute` aanroepen vervangen. Mijn PHPStorm geeft die binnen een project overigens niet aan als deprecated omdat ze daar in de vendor-map zitten.